src/account/ModuleManagerInternals.sol

```diff
@@ -162,8 +152,13 @@ abstract contract ModuleManagerInternals is IModuleManager {
         for (uint256 i = 0; i < length; ++i) {
             ManifestExecutionHook memory mh = manifest.executionHooks[i];
             EnumerableSet.Bytes32Set storage execHooks = _storage.selectorData[mh.executionSelector].executionHooks;
-            ModuleEntity hookFunction = ModuleEntityLib.pack(module, mh.entityId);
-            _addExecHooks(execHooks, hookFunction, mh.isPreHook, mh.isPostHook);
+            HookConfig hookFunction = HookConfigLib.packExecHook({
```

Comment:
Also here.

I've noticed that we're using `HookConfig hookConfig`, `HookConfig hookFunction`, and `HookConfig hook` throughout the files here. Let's stick to one?

Reply:
Good catch, will stick to `HookConfig hookConfig`.